File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/PossessoParticella.ts
import { BaseCodeDescr } from '../baseClass/baseCodeDescr';
import { TitoloDiPossesso } from '../metaschema/TitoloDiPossesso';
import { IntervalloTemporale } from './IntervalloTemporale';

export class PossessoParticella {
    codice: number;
    titolo_Di_Possesso: TitoloDiPossesso;
    validita: IntervalloTemporale;
    Area: number;
    flag_cancellazione: boolean;
    codice_particella: string;

    constructor() {
        this.flag_cancellazione = false;
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/PossessoParticella.cs
using AgronicaCoreModelsSTD.metaschema;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class PossessoParticella
    {
        public TitoloDiPossesso titolo_Di_Possesso { get; set; }
        public IntervalloTemporale validita { get; set; }
        public double Area { get; set; }
        public int codice { get; set; }
        public bool flag_cancellazione { get; set; }
        public string codice_particella { get; set; }

        public PossessoParticella(int codice)
        {
            this.codice = codice;
        }

        public PossessoParticella()
        {

        }

    }
}
